msfdb

```diff
@@ -920,6 +799,18 @@ def parse_args(args)
       @options[:db_pool] = m
     }
 
+    opts.on('--db-admin-pass PASS', "Database admin password (default: #{@options[:msf_db_admin_pass]})") { |p|
+      @options[:msf_db_admin_pass] = p
+    }
+
+    opts.on('--db-admin-username NAME', "Database admin username (default: #{@options[:msf_db_admin_username]}") { |p|
+      @options[:msf_db_admin_username] = p
+    }
+
+    opts.on('--standalone', 'Use a pre-existing database cluster for initialization') {
```

Comment:
Thoughts on consolidating this into just `--connection-string`? That way the user can configure completely arbitrary values, rather than the current hard coded `127.0.0.1` host and `postgres` database name 👀